test/yaml-rest-runner/src/main/java/org/elasticsearch/test/rest/yaml/ESClientYamlSuiteTestCase.java

```diff
@@ -278,11 +278,18 @@ public static Iterable<Object[]> createParameters(NamedXContentRegistry executea
         return createParameters(executeableSectionRegistry, null);
     }
 
+    /**
```

Comment:
We introduced yamlParameters here to allow overriding variables declared as e.g. @hdfs_port@ in yaml specs that we used to replace as part of the gradle build. We now need to do that as part of the test setup itself as we do not know e.g. the port at build time but at test time.